### PR TITLE
[BE-70] Fix for jenv 5.5, as no shortest version is available

### PR DESCRIPTION
**Pull request by lpusok**
### Checklist

- [x] I've read and followed the [Contribution Guidelines](https://github.com/bitrise-steplib/.github/blob/main/CONTRIBUTING.md)
- [x] `step.yml` and `README.md` is updated with the changes (if needed)

### Version

Requires a *PATCH* [version update](https://semver.org/)

### Context

With jenv 5.5 shortest version of Java (`11`) is no longer available on macOS. (https://github.com/jenv/jenv/issues/366) Added a fallback option.

### Changes

Will now try to select these Java versions:
11 : `11`, `11.0`
17: `17`, `17.0`

### CODE REVIEW

File: javasetter/javasetter.go [hunk 1]
```diff
@@ -64,18 +64,41 @@ func (j JavaSetter) SetJava(version JavaVersion) (Result, error) {
...
 
+func (j JavaSetter) tryToInstallOneOf(versions []string) error {
+	var err error
+
+	for _, version := range versions {
+		cmdJenv := j.cmdFactory.Create("jenv", []string{"global", string(version)}, nil)
+		j.logger.Printf("$ %s", cmdJenv.PrintableCommandArgs())
+
+		var output string
+		if output, err = cmdJenv.RunAndReturnTrimmedCombinedOutput(); err != nil {
```
[the remark this answers is not in the document]
Comment (lpusok) [reply javasetter/javasetter.go]: Thanks, simplified  this part.

**Review by fehersanyi-bitrise — DISMISSED** [review]
lgtm

**Review by ofalvai — APPROVED** [review]
Thanks, I hate it 😆 But if it's broken in jenv at the moment then let's fix it here.

File: javasetter/javasetter.go [hunk 2]
```diff
@@ -64,18 +65,37 @@ func (j JavaSetter) SetJava(version JavaVersion) (Result, error) {
 	}
 }
 
+func (j JavaSetter) tryToInstallOneOf(versions []string) error {
```
Comment (ofalvai) [inline javasetter/javasetter.go:68]: nit: I would add a link to the jenv issue tracker so that in the future we'll remember why this was needed.

**Review by ofalvai — APPROVED** [review]

**Pull request merged**